ui/app/services/user-settings.js

```diff
@@ -4,4 +4,5 @@ import localStorageProperty from 'nomad-ui/utils/properties/local-storage';
 export default class UserSettingsService extends Service {
   @localStorageProperty('nomadPageSize', 25) pageSize;
   @localStorageProperty('nomadLogMode', 'stdout') logMode;
+  @localStorageProperty('nomadTopoVizPollingNotice', true) showTopoVizPollingNotice;
```

Comment:
hmm I forgot this exists, I could have used it for the recommendation dismissal message 😯